### ISSUE TITLE
accessible design by default

### ISSUE THREAD

**Issue by duckinator** [duckie-talks-computers/duckie-talks-computers.github.io#2]
aka stop fucking burying accessibility options behind inaccessible interfaces (e.g., Slack's themes, the "accessible colors" setting on http://caniuse.com/, etc).
